### ISSUE TITLE
feature: --no-color flag

### ISSUE THREAD

**Issue by chrisallenlane** [cheat/cheat#489]
Implement a `--no-color` flag that can be used to override colorization settings. This can be useful when, for example, reading a cheatsheet into a `vim` buffer. (Currently, output read into the buffer will contain ANSI color codes if colorization is enabled.)

**Comment by exploide** [comment]
Maybe it's useful to adopt the behaviour of `grep --color=auto`, `ip -color=auto` and the other tools.

`--color=` can be `always`, `never` or `auto`.

The nice one is `auto`, which could be the default. It prints colored, but only when stdout is a tty. This means colors are magically suppressed when piping or redirecting to other processes or files. Maybe this also does the trick for your `vim` usecase?!

**Comment by chrisallenlane** [comment]
> The nice one is auto, which could be the default. It prints colored, but only when stdout is a tty.

Oh - now that you point it out, that is clearly the better pattern. I've just implemented this behavior in the `3.0.3` release. Thanks for the suggestion! :tada: 

> Maybe it's useful to adopt the behaviour of grep --color=auto, ip -color=auto and the other tools.

Currently, the only valid `color` values are `true` and `false`. That said, I might make that change in the (eventual) `cheat` v4. I want to run `cheat` v3 for a little while, see what we got wrong, and then make one more (hopefully final) backwards-incompatible release, where we get everything tidied up.

Example: I lament creating both `-d` and `-p`. Really, a "naked" `-p` should do the job of `-d`, which should itself be deprecated. That's not a show-stopper, though. We can clean it up later.

I'm closing this ticket in favor of the "auto" colorization behavior. Thanks again!
